lib/Parse/Snort/Strict.pm

```diff
@@ -0,0 +1,42 @@
+package Parse::Snort::Strict;
+use base qw(Parse::Snort);
+
+use strict;
+use warnings;
+use Carp qw(croak);
+use List::Util qw(any);
```

Comment:
I would update your Makefile.PL to include List::Util 1.33 (any is introduced in that one and CPAN testers will otherwise fail).